Automattic/wp-calypso
# Added disclaimer for products in beta.

#### Proposed Changes

* The soon-to-be-launched Jetpack Social Advanced plan is in Beta, we are pricing it at 180$ for the first year, but we are going to double the price after one year. We need to add this to our terms and conditions.

#### Testing Instructions



* Do `yarn start` to boot up your local calypso after checking out to this branch. 
* Go to `http://calypso.localhost:3000/checkout/jetpack/jetpack_social_advanced_yearly` and look at the newly added disclaimer. 
* 
<img width="705" alt="Screenshot 2023-02-02 at 15 05 58" src="https://user-images.githubusercontent.com/6594561/216287468-db32e65f-fda1-44d4-8a7c-91f8484fb5c3.png">


* Go to   `http://calypso.localhost:3000/checkout/jetpack/jetpack_social_basic_yearly` and make sure that the newly added disclaimer is gone since it's not applicable to the selected product. 

#### Pre-merge Checklist





Related to #
